styles/logo-file-icons.less

```diff
@@ -184,77 +174,26 @@
 
   [data-name$=".scala"]:before { .file-icon(scala); }
 
-  span[data-name=".npmrc"]:before,
-  span[data-name="npm-debug.log"]:before,
-  span[data-name=".npmignore"]:before,
-  div[data-name$="package.json"]:before,
-  span[data-name$="package.json"]:before { .file-icon(npm); }
+  [data-name=".npmrc"]:before,
+  [data-name="npm-debug.log"]:before,
+  [data-name=".npmignore"]:before,
+  [data-name="package.json"]:before { .file-icon(npm); }
 
-  span[data-name$="bower.json"]:before { .file-icon(bower); }
-  span[data-name^="composer."]:before { .file-icon(composer); }
+  [data-name="bower.json"]:before { .file-icon(bower); }
+  [data-name="composer.json"]:before { .file-icon(composer); }
 
-  span[data-name$="Gruntfile.js"]:before { .file-icon(grunt); }
-  span[data-name$="gruntfile.js"]:before { .file-icon(grunt); }
+  [data-name^="Gruntfile"]:before { .file-icon(grunt); }
```

Comment:
Change it to 
```
[data-name^="Gruntfile"]:before,
[data-name^="gruntfile"]:before { .file-icon(grunt); }
```
... just like you did with Gulpfile below to keep it consistent 😃